src/main/java/windycall/Ui.java

```diff
@@ -0,0 +1,64 @@
+package windycall;
+
+import java.util.List;
+
+/**
+ * Deals with interactions with the user
+ * Makes the background of chatBox more pretty
+ */
+public class Ui {
+
+    public Ui() {
+
+    }
+
+    public static void space() {
+        System.out.print("     ");
+    }
+
+    public static void line() {
+        space();
+        System.out.println("--------------------------------------------------");
+    }
```

Comment:
For method names, it is best to use an action verb. Can we change it to `printSpace` and `printLine` respectively?